nemo/collections/llm/recipes/llama3_70b.py

```diff
@@ -306,6 +306,7 @@ def finetune_recipe(
         recipe.peft.dim = 16
         recipe.peft.alpha = 32
         recipe.peft.target_modules = ['linear_qkv']
+        recipe.optim.config.use_distributed_optimizer = False
```

Comment:
is this because the extra communication makes distributed optimizer slower for lora?

Reply:
yes, there is also some memory overhead, and since the number of params is tiny, we can just disable